__tests__/fixtures1.blocks/BlockWithDeclaredMix/BlockWithDeclaredMix.js

```diff
@@ -0,0 +1,8 @@
+import { decl } from 'bem-react-core';
+
+export default decl({
+    block : 'BlockWithDeclaredMix',
+    mix() {
```

Comment:
`mix(val)` is like `mix = val` and `addMix(val)` is like `mix.push(val)` so you need to use `addMix()` if you want to add new mix and don't want to override old value